tests/e2e.sh

```diff
@@ -62,7 +62,7 @@ deploy_operator() {
 }
 
 run_test() {
-    $KUBECTL wait --for=condition=available --timeout=10m -n observatorium deploy/minio || ($KUBECTL get pods --all-namespaces && exit 1)
+    $KUBECTL wait --for=condition=available --timeout=10m -n minio deploy/minio || ($KUBECTL get pods --all-namespaces && exit 1)
```

Comment:
since we specify the namespaces, and part of the manifests in here have a different one, maybe we should skip the `-n` flag?

Reply:
ack, it's overconstrained right now